hack/local-up-karmada.sh

```diff
@@ -28,7 +32,7 @@ fi
 # make controller-manager image
 export VERSION="latest"
 export REGISTRY="swr.ap-southeast-1.myhuaweicloud.com/karmada"
-make images
+make images --directory="${SCRIPT_ROOT}"
```

Comment:
Wow, I didn't know the option before. Thanks~

Reply:
This option just for our requirement

```
make --help
Usage: make [options] [target] ...
Options:
  -C DIRECTORY, --directory=DIRECTORY
                              Change to DIRECTORY before doing anything.
```

Reply:
Got it!